torchvision/prototype/features/__init__.py

```diff
@@ -13,3 +14,5 @@
 )
 from ._label import Label, OneHotLabel
 from ._mask import Mask
+
+from ._dataset_wrapper import VisionDatasetFeatureWrapper  # usort: skip
```

Comment:
I've put it under `torchvision.prototype.features` for now, because I think that is the best place as long as the `features` are still in the prototype namespace. After roll-out, we could certainly also put this under `torchvision.datasets`

Reply:
Now under `torchvision.prototype.datapoints`, but the spirit is the same.

Reply:
We have decided to bikeshed over this later. This is tracked in #7092.